src/format.h

```diff
@@ -259,14 +259,18 @@ struct scoutfs_btree_block {
 	/* leaf blocks have a fixed size item offset hash table at the end */
 };
 
+#define BTREE_VALUE_ALIGN 8
```

Comment:
If we're adding this to format.h it should have a SCOUTFS_ prefix.  It's effectively a public API, it gets used in userspace too.